### ISSUE TITLE
E3681 Error with Valid HealthCheckPath and Matcher on AWS::ElasticLoadBalancingV2::TargetGroup

### ISSUE THREAD

**Issue by yamhs** [aws-cloudformation/cfn-lint#4095]
### CloudFormation Lint Version

1.33.1

### What operating system are you using?

Windows/Linux

### Describe the bug

`cfn-lint` reports `E3681 Additional properties are not allowed` for `HealthCheckPath` and `Matcher` in an `AWS::ElasticLoadBalancingV2::TargetGroup` resource, even though AWS documentation lists them as valid.

### Expected behavior

No E3681 errors for HealthCheckPath and Matcher.

### Reproduction template

```yaml
  MyTargetGroup:
    Type: AWS::ElasticLoadBalancingV2::TargetGroup
    Properties:
      Name: !Sub ${Env}-MyService-TG
      TargetType: lambda
      Targets:
        - Id: !FindInMap [!Ref Env, Services, Arn]
      HealthCheckPath: /services/health-check/ping
      HealthCheckEnabled: true
      HealthCheckIntervalSeconds: 300
      HealthCheckTimeoutSeconds: 5
      HealthyThresholdCount: 2
      UnhealthyThresholdCount: 2
      Matcher:
        HttpCode: 200-299
```

**Comment by kddejong** [comment]
Already fixed in #4094

**Comment by yamhs** [comment]
@kddejong Still seeing the same error with that template. On both versions 1.34.0 and 1.33.2
